client/src/routing/AppRoutes.tsx

```diff
@@ -1,13 +1,13 @@
 import { Route, Switch, Redirect, BrowserRouter } from 'react-router-dom';
 
-import { LoginPage } from '../Pages/LoginPage';
+import { RegisterPage } from '../Pages/RegisterPage';
 import { AppRoute } from './AppRoute.enum';
 
 export const AppRoutes = () => {
   return (
     <BrowserRouter>
       <Switch>
-        <Route path={AppRoute.Login} exact component={LoginPage} />
```

Comment:
Usuwamy całkowicie LoginPage, dlaczego?

Reply:
zamiastlogin jest register. W naszej aplikacji login i register mamy na tym samym pagu, przy czym login nie jest jeszcze wogole zrobione. Trzeba sie zdecydowac na jedna nazwe albo np. na startPage bo nie damy Login&RegisterPage :)

Reply:
Okey to w takim razie ta nazwa jest słaba, może authorizationPage?